conf/messages

```diff
@@ -201,11 +201,26 @@ whatIsOwed.allPenalties.interest = £{0} in interest on penalties
 calculation.title.lpp = Late payment penalty
 calculation.caption = {0} to {1}
 calculation.heading.lpp = Late payment penalty
-calculation.key1 = Penalty amount
```

Comment:
Can we please have 'additional' instead of 'add' in the key name , just to maintain consistency wrt previous 'additional' key names.